meson.build

```diff
@@ -22,6 +22,9 @@ else
 endif
 conf.set('ENABLE_USE_BUFFER_HEADER', get_option('USE_BUFFER_HEADER'))
 
+cc = meson.get_compiler('c')
```

Comment:
It is probably better to get the `cpp` compiler as we are cpp.
And as we already get the `cpp` compiler to get the size of `off_t`, we should probably factorize this and get the compiler only once.